src/components/LoginForm.jsx

```diff
@@ -11,6 +12,11 @@ const LoginForm = () => {
     value: "",
     isTouched: false,
   });
+
+  useEffect(() => {
+    document.title = "Login";
```

Comment:
change this to the component and not access JS Element

Reply:
Done